README.md

```diff
@@ -39,19 +36,85 @@ Report issues, ask questions, and provide suggestions using:
+
 ```bash
+# PyPi
+pip install daal4py
+```
+<details><summary>[Click to expand] ⚠️ Package limitations </summary>
+
+| OS / Python version     | **Python 36** | **Python 37**  | **Python 38**| **Python 39**|
+| :-----------------------| :-----------: | :------------: | :-----------:| :-----------:|
+|    **Linux**            |   ✔️         |     ✔️         |     ✔️      |      ✔️      |
+|    **Windows**          |   ✔️         |     ✔️         |     ✔️      |      ✔️      |
+|    **OsX**              |   ✔️         |     ✔️         |     ✔️      |      ❌      |
+
+#### ℹ️ Additional limitations
+
+PyPi channel contains of the latest release -
+[daal4py 2021.1](https://github.com/IntelPython/daal4py/releases/tag/2021.1).
+Has support Intel CPU and GPU.
+
+</details>
+
+### Anaconda Cloud - Conda-Forge channel
+[![Conda Version](https://img.shields.io/conda/vn/conda-forge/daal4py)](https://anaconda.org/conda-forge/daal4py)
+
+```bash
+# Anaconda Cloud from Conda-Forge channel
 conda install daal4py -c conda-forge
 ```
-or from Intel channel:
+<details><summary>[Click to expand] ⚠️ Package limitations </summary>
+
+| OS / Python version     | **Python 36** | **Python 37**  | **Python 38**| **Python 39**|
+| :-----------------------| :-----------: | :------------: | :-----------:| :-----------:|
+|    **Linux**            |   ✔️         |     ✔️         |     ✔️      |      ✔️      |
+|    **Windows**          |   ✔️         |     ✔️         |     ✔️      |      ✔️      |
+|    **OsX**              |   ❌         |     ❌         |     ❌      |      ❌      |
+
+#### ℹ️ Additional limitations
+
+Conda-Forge channel contains of the release -
+[daal4py 2020.3](https://github.com/IntelPython/daal4py/releases/tag/2020.3).
+Has support only Intel CPU.
+
+</details>
+
+### Anaconda Cloud - Intel channel
+[![Conda Version](https://anaconda.org/intel/daal4py/badges/version.svg)](https://anaconda.org/intel/daal4py)
+
 ```bash
+# Anaconda Cloud from Intel channel
 conda install daal4py -c intel
 ```
+
+<details><summary>[Click to expand] ⚠️ Package limitations </summary>
+
+| OS / Python version     | **Python 36** | **Python 37**  | **Python 38**| **Python 39**|
+| :-----------------------| :-----------: | :------------: | :-----------:| :-----------:|
+|    **Linux**            |   ❌         |     ✔️         |     ❌      |      ❌      |
+|    **Windows**          |   ❌         |     ✔️         |     ❌      |      ❌      |
+|    **OsX**              |   ❌         |     ✔️         |     ❌      |      ❌      |
+
+#### ℹ️ Additional limitations
+
+Intel channel contains of the latest release -
+[daal4py 2021.1](https://github.com/IntelPython/daal4py/releases/tag/2021.1).
+Has support Intel CPU and GPU.
+
+</details>
+
 You can [build daal4py from sources](INSTALL.md) as well.
 
-# Getting Started
-Core functioanlity of daal4py is in place Scikit-learn patching - Same Code, Same Behavior but faster execution. 
+# ⚡️ Getting Started
```

Comment:
guess that getting started should be higher. So person landing on page can understand what daal4py is.

Reply:
Do you propose move before `Installation`?

Reply:
Please also make it "Get Started" to be in line with our style guide

Reply:
done